recompose-composer/src/main/kotlin/recompose/composer/visitor/ComposingVisitor.kt

```diff
@@ -222,13 +222,29 @@ internal class ComposingVisitor : Visitor {
 
     override fun visitEditText(node: EditTextNode) {
         val modifier = ModifierBuilder(node)
-
+        val hintParameterValue = if (node.hint.isNotBlank()) {
+            ParameterValue.LambdaValue {
+                writeCall(
+                    name = "Text",
+                    parameters = listOf(
+                        CallParameter(name = "text", value = ParameterValue.StringValue(node.hint)),
+                        node.textColorHint?.let {
+                            CallParameter(name = "color", value = ParameterValue.ColorValue(it))
+                        },
+                    ),
+                    endLine = false
+                )
+            }
+        } else {
+            ParameterValue.EmptyLambdaValue
```

Comment:
According to the docs the default is `null`, I think?

Reply:
I guess ideally without `hint` this parameter wouldn't show up at all. If you turn it into `null` then in the parameter list you could do:

```Kotlin
hintParameterValue?.let { CallParameter(name = "label", value = it) }
```

`null` parameters will be filtered out automatically.

Reply:
Yes, the default value of label is `null`.